examples/seq2seq/run_seq2seq.py

```diff
@@ -25,10 +25,12 @@
 from dataclasses import dataclass, field
 from typing import Optional
 
+import nltk  # Here to have a nice missing dependency error message early on
```

Comment:
Is it in the `requirements.txt` from this example?

Reply:
Yes, all metrics related deps are in `requirements.txt`